doc/util/chpl2rst.py

```diff
@@ -215,8 +217,18 @@ def gen_rst(handle):
                     rstline = rstline[indentation:]
                 else:
                     rstline = rstline.lstrip(' ')
-
-            output.append(rstline)
+            # Special case for multi line comment in learnChapelInYMinutes
+            if(islearnChapelInYMinutes and ('multi-line comment' in rstline)):
```

Comment:
Minor nit: looks like we try to have a space between the `if` keyword and its conditions.  Also, I think we avoid using parentheses around the condition for Python style.